src/incubator/index.ts

```diff
@@ -6,8 +6,8 @@ export {default as TouchableOpacity2} from './TouchableOpacity2';
 export {default as WheelPicker, WheelPickerProps} from './WheelPicker';
 export {default as PanView, PanViewProps, PanViewDirections, PanViewDismissThreshold} from './panView';
 export {
-  default as TransitionAnimator,
-  TransitionAnimatorProps,
+  default as TransitionView,
+  TransitionViewProps,
   Direction,
   TransitionAnimationEndType
```

Comment:
Maybe export with TransitionView prefix so the types are connected to a component
Direction seem arbitrary, instead better name it `TransitionViewDirection`
And for `TransitionAnimationEndType` calling it `TransitionViewTransitionAnimationEndType` is a bit long, do you think we can shorten this type name a little?

Reply:
Regarding the `Direction`, it comes from `useHiddenLocation`, and I really think it should be used in `PanView` as well, so I'm not sure about `TransitionViewDirection`, will we want duplicate props?

How about `TransitionEndType`?

Reply:
So would `TransitionType` would be `TransitionViewTransitionType`? or `TransitionViewType`? 

And for `Direction`, will PanView export its own directions type?

Reply:
I thought just `TransitionType`, but if not then I guess it can be `TransitionViewTransitionType`; `TransitionViewType` sounds like a different thing (a type of the view and not the transition).

I'm not sure TBH, do you want to postpone and set it to `TransitionViewDirection` for now?

Reply:
Done

Reply:
@ethanshar
I thought I've renamed to `TransitionViewDirection`, I see in master it's not fixed, I'll fix in a future PR.